### PR TITLE
KV Sample fixes

### PR DESCRIPTION
**Pull request by amitbapat**
Fixed sample scripts
Fixed README

### CODE REVIEW

**Comment by amitbapat** [comment]
@pomortaz please review

**Comment by azurecla** [comment]
Hi __@amitbapat__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ambapat). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by cormacpayne** [comment]
LGTM

@pomortaz any comments?

File: src/KeyVault/Microsoft.Azure.KeyVault.Samples/README.md [hunk 1]
```diff
@@ -6,50 +6,15 @@
 
 ## Pre-requisites
 1. Visual Studio 2015 or Visual Studio 2013
-2. Azure SDK 2.8 
-3. Azure PowerShell version 1.0.0 or newer
+2. Azure SDK 2.9 
+3. Azure PowerShell version 2.1.0 or newer
```
Comment (pomortaz) [inline src/KeyVault/Microsoft.Azure.KeyVault.Samples/README.md]: The latest version of the PowerShell is 3.x

File: src/KeyVault/Microsoft.Azure.KeyVault.Samples/scripts/GetServiceConfigSettings.ps1 [hunk 2]
```diff
@@ -10,6 +10,13 @@ $resourceGroupName   = 'MyResourceGroupName'
 $applicationName     = 'MyAppName'
 $storageName         = 'MyStorageName'
 
+$vaultName           = 'demo9218'
+$resourceGroupName   = 'demo9218rg'
+$applicationName     = 'demo9218app'
+$storageName         = 'demo9218sa'
```
Comment (pomortaz) [inline src/KeyVault/Microsoft.Azure.KeyVault.Samples/scripts/GetServiceConfigSettings.ps1]: Use generic names and validate them.

**Review by pomortaz — CHANGES_REQUESTED** [review]

File: src/KeyVault/Microsoft.Azure.KeyVault.Samples/README.md (hunk 1 above)
Comment (amitbapat) [inline src/KeyVault/Microsoft.Azure.KeyVault.Samples/README.md]: Minimum version is 2.1, users can use any newer version.

**Review by pomortaz — APPROVED** [review]

**Pull request merged**